torch/csrc/jit/mobile/function.cpp

```diff
@@ -53,6 +53,17 @@ bool Function::append_operator(
   return true;
 }
 
+void Function::set_module_debug_info_list_size(size_t size) {
+  pc_to_module_debug_info_.resize(size);
+}
```

Comment:
I think it is ok to just do push_back. We dont expect to do this too often. We should aim to as less of a code as possible, unless needed. Less coding is better coding, I think.

Reply:
Yes, but I don't think we can do `push_back` here because the vector may be sparse, i.e. we don't have module info for every `pc` value.